src/containers/target-pane.jsx

```diff
@@ -76,6 +77,9 @@ class TargetPane extends React.Component {
     handleChangeSpriteY (y) {
         this.props.vm.postSpriteInfo({y});
     }
+    handleCloneSprite () {
```

Comment:
Looks good. Tested it and the button click triggers this function, as expected.

One thing though: Do we want to merge this right away and work on implementing this function separately, or just finish implementing this too and merge later?